src/main/java/com/google/sps/data/GovParameters.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
+/** A class, classified as GovParameters */
+public class GovParameters {
+  @SerializedName(value = "Site_Id")
+  private final int siteId;
+  @SerializedName(value = "Value")
+  private final double aqi; // AQI
+  
+  //No arg constructor to help differentiate a iniatised class with invalid values 
+  public GovParameters() {
+    this.siteId = -1;
+    this.aqi = -1;
+  }
+
+  public GovParameters(int siteId, double aqi) {
+    this.siteId = siteId;
+    this.aqi = aqi;
+  }
+
+  //Overriding the toString() method  
```

Comment:
This comment should be replaced with an `@Override` annotation, see https://www.geeksforgeeks.org/overriding-in-java/